src/main/scala/HairMan.scala

```diff
@@ -0,0 +1,3 @@
+case class HairMan (name: String, glasses: Boolean, facialHair: Boolean, hairColour: String, hat: Boolean, eyeColour: String) extends Male {
```

Comment:
Consider again whether we need this + having hair prompts having a different case class altogether?

Reply:
This makes complete sense. We decided to stick with adding hair properties directly rather than creating a separate case class.